test/controllers/works_controller_test.rb

```diff
@@ -35,12 +35,14 @@
 
   describe "index" do
     it "succeeds when there are works" do
+      perform_login
```

Comment:
One problem with these tests is that you don't have a set of tests for **guest users**.  So you don't have tests that assure that guest users can't do specific things like view a show page, or delete a work.